python/test/t_NearestPointChecker_std.py

```diff
@@ -1,36 +0,0 @@
-#! /usr/bin/env python
```

Comment:
Why removing this unit test?

Reply:
Because we removed the class from the python API. It is of no use for the User. It is used for the StrongMaximumTest only .